models/DataObject/ClassDefinition/Data/ReverseObjectRelation.php

```diff
@@ -208,6 +244,28 @@ public function isOptimizedAdminLoading(): bool
     {
         return true;
     }
+
+    protected function enrichDataRow($object, array $params, ?string &$classId, &$row = [], string $srcCol = 'src_id')
+    {
+        if (!$row) {
+            $row = [];
+        }
+
+        if ($object instanceof DataObject\Concrete) {
+            $row['dest_id'] = $object->getId();
+            $row['ownertype'] = 'object';
+
+            $classId = $this->getOwnerClassId();
+        }
+    }
+
+    public function preGetData($container, $params = [])
+    {
+        if($container instanceof DataObject\LazyLoadedFieldsInterface) {
+            $container->unmarkLazyKeyAsLoaded($this->getName());
+        }
+        return parent::preGetData($container, $params);
```

Comment:
```suggestion
        return $this->load($container);
```
Since reverse relations are only allowed in the object context (not in localized fields, field collections, object bricks, it should be fine to just return the loaded data. No need to store it on the property or care about lazy loading - let's keep things simple 😉